engine/src/main/java/org/terasology/engine/TerasologyEngine.java

```diff
@@ -315,6 +318,8 @@ private void initAssets() {
                 (AssetFactory<BehaviorTree, BehaviorTreeData>) BehaviorTree::new, false, "behaviors");
         assetTypeManager.registerCoreAssetType(UIElement.class,
                 (AssetFactory<UIElement, UIData>) UIElement::new, "ui");
+        assetTypeManager.registerCoreAssetType(Translation.class,
```

Comment:
I'ld suggest this should be moved into the I18nSubsystem too (I moved the reset of the asset type initialisations into subsystems in my engine lifecycle rework branch).